generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -24,6 +31,22 @@ export const newTestModelGenerator: CodeGenerator = {
     });
     const modelCodename = snakeCase(modelName).replaceAll('_', '-');
 
+    const { modelOwningService } = await prompts({
```

Comment:
Since the GraphQL types from the services are prefixed, we need to ask about which service provide the type the developer wants to generate in order to correctly build the `import` statements in the templates.